docs/typescript/reference/client.md

```diff
@@ -0,0 +1,172 @@
+---
+sidebar_position: 25
+title: DBOS Client
+description: DBOS Client reference
+---
+
+`DBOSClient` provides a programmatic way to interact with your DBOS application from external code.
+`DBOSClient` includes methods similar to [`DBOS`](./transactapi/dbos-class.md)
+that make sense to be used outside of a DBOS workflow or step, such as `enqueueWorkflow` or `getEvent`.
+
+:::note 
+`DBOSClient` is included in the `@dbos-inc/dbos-sdk` package, the same package that used by DBOS applications.
+Where DBOS applications use the [static `DBOS` class](./transactapi/dbos-class.md),
+external applications use the `DBOSClient` class instead.
+:::
+
+### class DBOSClient
+
+```ts
+interface EnqueueOptions {
+    workflowName: string;
+    workflowClassName: string;
+    queueName: string;
+    appVersion?: string;
+    workflowID?: string;
+}
+
+class DBOSClient {
+    static create(databaseUrl: string, systemDatabase?: string): Promise<DBOSClient>;
+    destroy(): Promise<void>;
+    async enqueue<T extends (...args: any[]) => Promise<any>>(
+        options: EnqueueOptions,
+        ...args: Parameters<T>
+    ): Promise<WorkflowHandle<Awaited<ReturnType<T>>>>
+    enqueue<T extends unknown[]>(options: EnqueueOptions, ...args: T): Promise<void>;
```

Comment:
Why two enqueue here?

Reply:
When I changed enqueue to return a WF Handle, I pasted in the new signature and forgot to delete the old one. Will remove the old signature